.travis.yml

```diff
@@ -1,32 +1,50 @@
 language: rust
+
+# hopefully prevent the cache from becoming huge - https://levans.fr/rust_travis_cache.html
+before_cache:
+  - rm -rf /home/travis/.cargo/registry
 cache:
   directories:
     - "$HOME/.cargo"
     - "$HOME/.ivy2/cache"
     - "$HOME/.sbt/boot/"
     - "$HOME/bin"
-# hopefully prevent the cache from becoming huge - https://levans.fr/rust_travis_cache.html
-before_cache:
-  - rm -rf /home/travis/.cargo/registry
-rust:
-  - 1.34.1
+
+# build matrix
+rust: 1.35.0
+matrix:
+  include:
+    - name: "Linux"
+      os: linux
+    - name: "OSX"
+      os: osx
+
 branches:
   only:
     - master
+
 before_install:
   - openssl aes-256-cbc -K $encrypted_487cf28eaba8_key -iv $encrypted_487cf28eaba8_iv
     -in tests/src/test/resources/service-keys.conf.enc -out tests/src/test/resources/service-keys.conf -d
 install:
   - "~/bin/sbt --help || curl -Ls https://git.io/sbt > ~/bin/sbt && chmod 0755 ~/bin/sbt"
+
+# tests
```

Comment:
What is this `tests` comment for?

Reply:
It was previously `# run tests`. Just had this discussion with Colt. When I was reading this file and thinking about it, I needed to delineate what the different pieces of the file were doing. First that was moving things around to be whitespace grouped by the task they're accomplishing. With the top level `deploy` command and the caching that was now clear. I added `build matrix` to the section of `rust` and `matrix` because it's actually the combination of the two that creates the build matrix. I added `run tests` because that's what that section was there for and `script`/`before_script` didn't tell me that. Reduced it to just `tests` because Colt called out that the before scripts line specifically that it's on doesn't run the tests.

Reply:
Updated again.